### PR TITLE
Add password requirements for registration

### PR DESCRIPTION
**Pull request by shadowhand**
Refs WEB-2781

### CODE REVIEW

File: source/index.md.erb [hunk 1]
```diff
@@ -230,7 +230,12 @@ Key | Description
 <% param "subdomain", "string" do %> The subdomain of this user.<%end%>
 <% param "account_id", "string(optional)" do %> The account id of this user.<%end%>
 
+Please note that all passwords must be:
 
+- at least 10 characters long
+- may not contian the email address
```
Comment (ameech) [inline source/index.md.erb]: contain

**Comment by ameech** [comment]
Looks good after that fix.

**Comment by th3fallen** [comment]
looks good after spelling.

File: source/index.md.erb [hunk 2]
```diff
@@ -230,7 +230,12 @@ Key | Description
 <% param "subdomain", "string" do %> The subdomain of this user.<%end%>
 <% param "account_id", "string(optional)" do %> The account id of this user.<%end%>
 
+Please note that all passwords must be:
 
+- at least 10 characters long
+- may not contain the email address
```
Comment (shadowhand) [inline source/index.md.erb]: Fixed. Pushing it.

**Pull request merged**